Airports/EGLK/SMR/Region.txt

```diff
@@ -0,0 +1,414 @@
+N051.19.19.303 W000.50.55.299
+N051.19.19.071 W000.50.54.975
+N051.19.18.941 W000.50.54.609
+N051.19.18.889 W000.50.52.210
+N051.19.18.841 W000.50.50.730
+N051.19.18.860 W000.50.50.174
+N051.19.18.907 W000.50.49.872
+N051.19.19.656 W000.50.47.148
+N051.19.20.229 W000.50.46.641
+N051.19.20.481 W000.50.46.772
+N051.19.20.879 W000.50.47.225
+N051.19.21.036 W000.50.48.258
+N051.19.20.279 W000.50.54.778
+N051.19.19.936 W000.50.55.116
+
+;Runway
+REGIONNAME Blackbushe
+smrTaxiway N051.19.30.710 W000.50.27.272
+N051.19.32.110 W000.50.28.011
+N051.19.21.866 W000.51.17.001
+N051.19.20.503 W000.51.16.245
+
+;Buildings
+REGIONNAME Blackbushe
+terminal N051.19.22.295 W000.50.29.270
+N051.19.21.798 W000.50.29.147
+N051.19.21.802 W000.50.28.923
+N051.19.21.910 W000.50.28.944
+N051.19.22.003 W000.50.27.633
+N051.19.22.294 W000.50.27.699
+N051.19.22.575 W000.50.27.729
+N051.19.22.519 W000.50.28.660
+N051.19.22.249 W000.50.28.571
+N051.19.22.164 W000.50.29.047
+N051.19.22.319 W000.50.29.066
+REGIONNAME Blackbushe
+terminal N051.19.23.784 W000.50.26.112
+N051.19.23.922 W000.50.25.523
+N051.19.24.128 W000.50.25.624
+N051.19.24.194 W000.50.25.398
+N051.19.24.313 W000.50.25.473
+N051.19.24.110 W000.50.26.704
+N051.19.23.729 W000.50.26.491
+REGIONNAME Blackbushe
+terminal N051.19.21.766 W000.50.26.412
+N051.19.22.029 W000.50.26.469
+N051.19.22.081 W000.50.26.285
+N051.19.22.463 W000.50.26.405
+N051.19.22.488 W000.50.26.536
+N051.19.22.684 W000.50.26.605
+N051.19.22.801 W000.50.26.875
+N051.19.22.955 W000.50.26.915
+N051.19.23.061 W000.50.26.746
+N051.19.23.004 W000.50.27.309
+N051.19.22.836 W000.50.27.122
+N051.19.22.627 W000.50.27.355
+N051.19.22.397 W000.50.27.284
+N051.19.22.356 W000.50.27.457
+N051.19.21.688 W000.50.27.328
+REGIONNAME Blackbushe
+terminal N051.19.21.852 W000.50.24.313
+N051.19.22.149 W000.50.24.407
+N051.19.22.093 W000.50.25.014
+N051.19.21.793 W000.50.24.965
+REGIONNAME Blackbushe
+terminal N051.19.25.405 W000.50.25.848
+N051.19.25.682 W000.50.26.007
+N051.19.25.550 W000.50.26.686
+N051.19.25.270 W000.50.26.508
+REGIONNAME Blackbushe
+terminal N051.19.21.797 W000.50.21.858
+N051.19.21.585 W000.50.21.772
+N051.19.21.595 W000.50.21.549
+N051.19.21.608 W000.50.21.201
+N051.19.21.512 W000.50.21.137
+N051.19.21.545 W000.50.20.813
+N051.19.21.668 W000.50.20.861
+N051.19.21.674 W000.50.20.652
+N051.19.21.497 W000.50.20.614
+N051.19.21.548 W000.50.20.017
+N051.19.21.711 W000.50.20.071
+N051.19.21.721 W000.50.19.974
+N051.19.22.064 W000.50.20.030
+N051.19.21.938 W000.50.21.612
+N051.19.21.826 W000.50.21.603
+
+;Numbers and Helipad
+REGIONNAME Blackbushe
+smrWhite N051.19.31.443 W000.50.28.381
+N051.19.31.490 W000.50.28.167
+N051.19.31.378 W000.50.28.105
+N051.19.31.405 W000.50.27.985
+N051.19.31.566 W000.50.28.066
+N051.19.31.466 W000.50.28.540
+N051.19.31.353 W000.50.28.495
+N051.19.31.320 W000.50.28.616
+N051.19.31.438 W000.50.28.673
+N051.19.31.410 W000.50.28.800
+N051.19.31.245 W000.50.28.720
+N051.19.31.335 W000.50.28.320
+REGIONNAME Blackbushe
```

Comment:
Any use in adding further labels to describe what _exactly_ these are for? If not, then there is no need to repeat the REGIONNAME Blackbushe and smrWhite entries

Reply:
If we don't repeat the Regionname etc-  the end of one region becomes the start of the next - and the building, or number or whatever all merges into one messy area interconnected